### ISSUE TITLE
DBException appears only with 'top' parameter present

### ISSUE THREAD

**Issue by frabarz** [tesseract-olap/tesseract#265]
I got this error while trying to load [this query](https://api.oec.world/tesseract/data.jsonrecords?cube=trade_i_baci_a_02&drilldowns=Year,Section&measures=Trade+Value,Quantity&rca=Year,Section,Trade+Value&top=3,Section,Trade+Value,asc):
```
Server error: `ERROR DB::Exception (47): DB::Exception: Missing columns: 'final_m0' while processing query: '<SQL QUERY>', 
required columns: 'final_m0' 'final_m1' 'final_m2' 'rca' 'section_id_HS_Product' 'section_name_HS_Product', 'year_Year'
source columns: 'final_m1'` 'final_m2' 'rca' 'section_id_HS_Product' 'section_name_HS_Product' 'year_Year' 
```

This query however works correctly [if the `top` parameter is omitted](https://api.oec.world/tesseract/data.jsonrecords?cube=trade_i_baci_a_02&drilldowns=Year,Section&measures=Trade+Value,Quantity&rca=Year,Section,Trade+Value).

It's worth investigating if there's a missing column, or if the problem comes from a bug on tesseract.

**Comment by jspeis** [comment]
@frabarz is this now fixed?

**Comment by frabarz** [comment]
Nope, but the issue with this URL changed. Now both queries, with and without `top` parameter, are returning a float point value in the `Year` key.
